src/components/Page/tests/Page.test.tsx

```diff
@@ -170,7 +170,7 @@ describe('<Page />', () => {
       };
 
       const {restore: restoreTitleBarCreateMock} = mockTitleBarCreate();
-      AppBridgeButton.create = jest.fn().mockReturnValue(buttonMock);
+      jest.spyOn(AppBridgeButton, 'create').mockReturnValue(buttonMock);
```

Comment:
^

Reply:
It looks like define restoration - see line 178